src/SFML/Graphics/Text.cpp

```diff
@@ -30,40 +30,81 @@
 #include <SFML/Graphics/Text.hpp>
 #include <SFML/Graphics/Texture.hpp>
 
+#if defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wold-style-cast"
+#elif defined(__GNUC__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wold-style-cast"
```

Comment:
Why is the use of `SYSTEM` include dirs not sufficient to silence these 3rd party warnings?

Reply:
Now that CMake was updated, can you implement this with the `SYSTEM` property? And/or explain what the necessary change is, because I haven't fully understood it.

Reply:
Done.